kit/scrolls/ios/UIKitScrollView.m

```diff
@@ -52,6 +52,12 @@ - (void)didMoveToWindow {
     }
 }
 
+- (void)layoutSubviews {
```

Comment:
This is actual fix of auto insets calculation, as just to measure them on mount isn't enough, we have to take layout changes into account too

Reply:
👍🏻